index.html

```diff
@@ -51,5 +52,183 @@ <h3>LET’S CONNECT</h3>
         </ul>
       </section>
     </div>
+    <section class="grid-container">
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h1>Tonic</h1>
```

Comment:
kindly use an `h2` or `h3` tag for the headings of each of your projects here,as `h1` tag is only to be used for the first or main title in a webpage and for semantic reasons the hierarchy of header tags is from `h1-h6` in that order, since you have an `h2` tag above you cannot have another `h1` tag here 👍